python/lsst/ts/xml/data/sal_interfaces/Electrometer/Electrometer_Events.xml

```diff
@@ -57,7 +57,7 @@
   <SALEvent>
     <Subsystem>Electrometer</Subsystem>
     <EFDB_Topic>Electrometer_logevent_intensity</EFDB_Topic>
-    <Description>Intensity of light measured with the photo-diode in the unit previously configured</Description>
+    <Description>Intensity of light measured with the photo-diode in the unit previously configured. NOT IMPLEMENTED </Description>
```

Comment:
Is this still true? I thought that intensity event was publishing albeit with some saturation issues.

Reply:
I think it only publishes at 1 Hz during a scan... 
unsure about saturation..

Reply:
Ok good to know. I hadn't see that yet. I will modify.